jsk_perception/src/apply_mask_image.cpp

```diff
@@ -53,6 +53,7 @@ namespace jsk_perception
     pnh_->param("mask_black_to_transparent", mask_black_to_transparent_, false);
     pnh_->param("use_rectified_image", use_rectified_image_, true);
     pnh_->param("queue_size", queue_size_, 100);
+    pnh_->param("max_interval_duration", max_interval_duration_, ros::DURATION_MAX.sec);
```

Comment:
toSec() might be better.
```suggestion
    pnh_->param("max_interval_duration", max_interval_duration_, ros::DURATION_MAX.toSec());
```
https://github.com/ros/roscpp_core/blob/noetic-devel/rostime/include/ros/duration.h#L92

Reply:
Thank you.